### PR TITLE
feat: Implement a new bar chart design

### LINKED ISSUE #1084: Update a bar chart UI to match the new design
**Description**
Currently, the UI of the bar chart does not look like the new design.

**Acceptance Criteria**
- [ ] A new bar chart UI matches the updated design.

**Additional context**
We want to change the bar chart without touching the range of dates.

Project Bar

### PR DESCRIPTION
**Pull request by solid-tetianamovlian**
Fixes #1084 

## Checklist

- [x] Documented
- [x] Component tested

### CODE REVIEW

File: metrics/web/lib/dashboard/presentation/widgets/build_result_bar_graph.dart [hunk 1]
```diff
@@ -69,36 +69,44 @@ class _BuildResultBarGraphState extends State<BuildResultBarGraph> {
       buildResults: _barsData,
     );
 
-    return Row(
-      crossAxisAlignment: CrossAxisAlignment.end,
-      children: <Widget>[
-        Flexible(
+    return Column(
```
Comment (solid-vladlaba) [inline metrics/web/lib/dashboard/presentation/widgets/build_result_bar_graph.dart:72]: It seems like the `Align` widget with the alignment `bottomCenter` would be enough for now as we don't introduce the date range.
```suggestion
    return Align(
```

**Review by solid-vladlaba — CHANGES_REQUESTED** [review]
Well done, @solid-maxim @solid-tetianamovlian! I've added a few small suggestions below - please let me know what you think!

File: metrics/web/test/dashboard/presentation/widgets/metrics_table_row_test.dart [hunk 2]
```diff
@@ -121,19 +121,6 @@ void main() {
       },
     );
 
-    testWidgets(
```
[the remark this answers is not in the document]
Comment (solid-tetianamovlian) [reply metrics/web/test/dashboard/presentation/widgets/metrics_table_row_test.dart:124]: This test case is duplicate.

**Review by solid-vladlaba — APPROVED** [review]
LGTM, thanks @solid-tetianamovlian @solid-maxim !:rocket:

**Pull request merged**